spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb

```diff
@@ -3,54 +3,38 @@
 require 'spec_helper_acceptance'
 
 describe 'redis::sentinel' do
-  redis_name = case fact('osfamily')
-               when 'Debian'
-                 'redis-server'
-               else
-                 'redis'
-               end
-
-  it 'runs successfully' do
-    pp = <<-EOS
-    class { 'redis::sentinel':
-      master_name      => 'mymaster',
-      redis_host       => '127.0.0.1',
-      failover_timeout => 10000,
-    }
-    EOS
-
-    apply_manifest(pp, catch_failures: true)
-    apply_manifest(pp, catch_changes: true)
-  end
-
-  describe package(redis_name) do
-    it { is_expected.to be_installed }
+  redis_name = fact('osfamily') == 'Debian' ? 'redis-server' : 'redis'
```

Comment:
osfamily is a legacy fact; but this might be alright of we have more legacy facts that we all fix in one PR